inventory/migrations/0013_auto_20200418_1425.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 3.0.3 on 2020-04-18 11:25
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
I can see you making lots of modification on the models. Please squash all the migrations into just one
or delete them and just have 1 migration (this is possible as the migration only were applied to your local DB)